### DISCUSSION TITLE
Disabling LSP Diagnostics for a Specific Line

### DISCUSSION THREAD

**Discussion by Koda777** [nvimdev/lspsaga.nvim#1266] [Q&A] [answered]
Hi,

The reason for this is that I'm dealing with a scenario where the LSP diagnostics are reporting an issue on a line that is valid in the context of my project's requirements. While I understand the importance of diagnostics, I would like to know if there's a way to suppress or disable diagnostics for just this line without affecting the rest of the codebase.

Thank you for your assistance and support. Looking forward to your guidance on how to handle this situation effectively.

**Comment by glepnir** [comment]
you mean disable the sign and virtual text ?

**Reply by Koda777** [reply]
yup exactly

**Comment by glepnir** [comment, marked as answer]
this is not relate lspsaga. but you can try

```lua
vim.api.nvim_create_autocmd('BufEnter', {
  pattern = your special file pattern
   callback = function(opt)
      vim.diagnostic.disable(opt.buf)
   end
})
```

**Reply by Koda777** [reply]
Thank you
